apps/condo/domains/user/schema/UserExternalIdentity.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Generated by `createschema user.UserExternalIdentity`
+ */
+
+const { get } = require('lodash')
+const { Text, Select, Relationship } = require('@keystonejs/fields')
+const { Json } = require('@open-condo/keystone/fields')
+const { GQLListSchema, getById } = require('@open-condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+
+const { IDP_TYPES, RESIDENT } = require('@condo/domains/user/constants/common')
+
+const access = require('@condo/domains/user/access/UserExternalIdentity')
+
+const UserExternalIdentity = new GQLListSchema('UserExternalIdentity', {
+    schemaDoc: 'Individual user external identity link. Used primarily for authorization and linking to external identity provider purposes. Think of `User` as a technical entity, not a business actor.',
+    fields: {
+        user: {
+            schemaDoc: 'Link to user',
+            type: Relationship,
+            ref: 'User',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.PROTECT' },
+        },
+        identityId: {
+            schemaDoc: 'External identity id. The value of this field should be populated from an external identity provider',
+            type: Text,
+            isRequired: true,
+        },
+        identityType: {
+            schemaDoc: 'The type of external identity that was a source for this link',
+            type: Select,
+            dataType: 'enum',
+            options: IDP_TYPES,
+            isRequired: true,
+        },
+        meta: {
```

Comment:
It looks like the SBBOL and SBID meta should be saved here

Reply:
Yes, this is a place for external identity meta